src/macros.rs

```diff
@@ -2,12 +2,10 @@
 //!
 //! We're avoiding static initializers, so we can't have things like string
 //! literals. Replace the standard assert macros with simpler implementations.
-
-// TODO: Wire this up to stderr.
 #[allow(dead_code)]
 #[doc(hidden)]
 pub fn print(message: &[u8]) {
-    crate::bindings::stderr::print(message)
+    let _ = unsafe { crate::bindings::streams::write(crate::get_stderr_stream(), message) };
```

Comment:
Would it be possible to call the preopens function to re-learn the stderr number here to avoid needing a new global to store this?

Reply:
I don't think so - when we move to resources, get_stdio will transfer ownership of the resources, so we'll have to add a restriction that get_stdio traps if called multiple times.